### ISSUE TITLE
Usage of @click on Steps breaks forms.

### ISSUE THREAD

**Issue by CromonMS** [pulsardev/vue-tour#33]
As per previous issue I am using this is a way of explaining a form in detail.

Clicking on Skip Tour submits my <form></form> and refreshes the page with the form on effectively adding a very long string to the address bar.

I propose to use @click.prevent on the buttons to stop this behaviour.

```
<button @click.prevent="stop" v-if="!isLast" class="v-step__button">Skip tour</button>
<button @click.prevent="previousStep" v-if="!isFirst" class="v-step__button">Previous</button>
<button @click.prevent="nextStep" v-if="!isLast" class="v-step__button">Next</button>
<button @click.prevent="stop" v-if="isLast" class="v-step__button">Finish</button>
```

**Comment by CromonMS** [comment]
After a little experiment, it could also work to use <a> tag instead of buttons. Down to your preference really.

**Comment by Outpox** [comment]
Merged trough your PR in #32 
Thanks you! :-)
